infra/dockerfile.train_stepstoreproduce

```diff
@@ -0,0 +1,3 @@
+FROM mozilla/bugbug-base:latest
+
+CMD bugbug-train stepstoreproduce
```

Comment:
Can you try if this command works for you locally? You'll probably need to update some other file in order for it to run successfully.

Reply:
How do I run these tasks locally?

Reply:
For the purpose of this PR, you can just run this command locally. bugbug-train is defined in setup.py, and it corresponds to scripts.trainer:main.
So you can run `python3 -c "from scripts import trainer; trainer.main()" stepstoreproduce`.

Reply:
Yes, this needed to be added to the trainer.py script too. I've added this now.

Reply:
Thanks. We'll need to wait for an image on Docker Hub to be created for this.

Reply:
The image has been added, so we can now merge this.